packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -641,14 +719,18 @@ export default class ConversationService {
     payloadBundle: PayloadBundleOutgoingUnsent
   ): Promise<PayloadBundleOutgoing> {
     switch (payloadBundle.type) {
-      case GenericMessageType.ASSET: {
+      case GenericMessageType.ASSET:
+      case GenericMessageType.IMAGE: {
         if (payloadBundle.content) {
           if ((payloadBundle.content as ImageAssetContent).image) {
             return this.sendImage(conversationId, payloadBundle);
           }
-          throw new Error(`No send method implemented for sending other assets than images.`);
+          if ((payloadBundle.content as FileAssetContent).asset) {
+            return this.sendFile(conversationId, payloadBundle);
+          }
+          throw new Error(`No send method implemented for this kind of asset.`);
         }
-        throw new Error(`No send method implemented for "${payloadBundle.type}" without content".`);
+        throw new Error(`No send method implemented for "${payloadBundle.type}" without content.`);
```

Comment:
Maybe the text `Payload of type "${payloadBundle.type}" does not have a "content" property"` is more specific?